src/cljs_time/coerce.cljs

```diff
@@ -50,7 +50,7 @@
 (defn to-epoch
   "Convert `obj` to Unix epoch."
   [obj]
-  (some-> obj to-long (/ 1000)))
+  (some-> obj to-long (/ 1000) int))
```

Comment:
A quick performance check on `1000000` iterations appears nearly identical either way we go.